src/tests/Interop/COM/Activator/Program.cs

```diff
@@ -259,11 +279,23 @@ static int Main(string[] doNotUse)
         {
             try
             {
+                bool builtInComDisabled = false;
+                var comConfig = AppContext.GetData("System.Runtime.InteropServices.BuiltInComInterop.IsSupported");
```

Comment:
Maybe use `AppContext.TryGetSwitch` instead? Seems more in line with the intent and you wouldn't need to do the conversion to a bool yourself.

Reply:
Does this API work with the `RuntimeHostConfigurationOption`? A quick doc search doesn't seem to show that

Reply:
I would look at the source:
https://github.com/dotnet/runtime/blob/d07f9111c57fab97aa135bebb2687471a8bfc5bf/src/libraries/System.Private.CoreLib/src/System/AppContext.cs#L105

The product itself is currently going through `TryGetSwitch` to get feature switch values (so it better work).